### DISCUSSION TITLE
Issue with codedFixed command

### DISCUSSION THREAD

**Discussion by Vihangajae** [blueCFD/Core#228] [Show and tell]
When I tried cfd project with codedFixed values for velocity profile it shows error called
"" FOAM FATAL IO ERROR"" why that happens? I can only work with fixed values.

**Comment by Vihangajae** [comment]
![7312f903-0f7c-475e-87bb-ae0eeea224ca](https://user-images.githubusercontent.com/92616496/137572435-2e9e9b69-539b-4167-9c55-b81cc05739f4.jpg)
![24567977-6538-4ff8-87ad-3da0bd4510ed](https://user-images.githubusercontent.com/92616496/137572439-b660bcd3-8f3c-4f46-b2bc-f753ca296849.jpg)

**Comment by Vihangajae** [comment]
This is my code and error

**Comment by wyldckat** [comment]
Quick answer @Vihangajae: Sorry for the delay, but the error message you've provided does not seem to be complete.

You should have gotten something like this:
![image](https://user-images.githubusercontent.com/877290/138857246-0877f8f8-01a3-40c5-8974-6fb042894891.png)


So above the FATAL error message, it describes what went wrong.
